src/utils/Authenticate.ts

```diff
@@ -27,13 +31,13 @@ async function ensureAuthenticated(
 ) {
   const bearerToken = parseToken(req, res);
 
-  if (bearerToken === '') return next(true);
```

Comment:
I forgot to remove these guys in the last PR. They were for debugging and print true when authentication fails.